bot/bot.py

```diff
@@ -1,28 +1,26 @@
-import asyncio
-import logging
-import os
-import sys
-import time
-import traceback
-import aiohttp
+from fred_core_imports import *
 from cogs import commands, crashes, dialogflow, mediaonly, webhooklistener, welcome, levelling
-import discord
-import discord.ext.commands
-import sqlobject as sql
-from libraries import createembed, helper
+from libraries import createembed, common
+
 from logstash_async.handler import AsynchronousLogstashHandler
-import config
+import sqlobject as sql
+from dotenv import load_dotenv
+import nextcord.ext.commands
+
+
+load_dotenv()
```

Comment:
What's this?

Reply:
Loads a .env file just in case. I know docker is *supposed* to but it's a failsafe.